python/3.8/poetry/Dockerfile

```diff
@@ -13,6 +13,8 @@ ENV \
     PYTHONDONTWRITEBYTECODE=1
 
 # install poetry
+# before that make sure that we fail on pipe failurese (see https://github.com/hadolint/hadolint/wiki/DL4006)
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
```

Comment:
Nice! Did not know that.

Reply:
Me either, `hadolint` gave me the hint (: